scala_akka_bench/.gitignore

```diff
@@ -0,0 +1,8 @@
+.bloop/
```

Comment:
let's omit those ignore files, they are not really needed, especially the IDE-related ones.